packages/teams-js/test/private/remoteCamera.spec.ts

```diff
@@ -375,6 +440,21 @@ describe('remoteCamera', () => {
 
     Object.values(FrameContexts).forEach(context => {
       if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`remoteCamera.registerOnCapableParticipantsChangeHandler should throw error when remoteCamera is not supported. context : ${context}`, async () => {
+          await utils.initializeWithContext(context);
+          utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
+          let CapableParticipants: remoteCamera.Participant[];
```

Comment:
nit: lower case "c" in "capableParticipants" for consistency.

Reply:
Fixed in the latest commit.